frontend/src/fixes/global-styling.scss

```
@@ -0,0 +1,15 @@
+// Extensions run in an isolated iframe and miss the global `box-sizing:
+// border-box` reset the Marketplace host applies. Without it, SDK components
+// (Grid, List, Wizard…) overflow their containers by their border width.
+//
```

Stylelint (17.12.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)